swag/everywhere_tour_guide.py

```diff
@@ -1,42 +1,70 @@
+# everywhere_tour_guide.py
 import anthropic
-import os
 import logging
-from typing import  AsyncGenerator
+from typing import AsyncGenerator
+import base64
 
-from swag.assistant import Assistant 
-from swag.tools import SearchInternet, ReadWebsite, SearchForNearbyPlacesOfType
+from swags.assistant import Assistant
```

Comment:
This should be swag

Reply:
[Fixed imports](https://github.com/team-swag-dhst/llm-agents-berkeley/pull/11/commits/9d002a9af28594d7948c59f0da95f7f408c734b7)